hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestContainerAllocation.java

```diff
@@ -169,6 +172,7 @@ public void testContainerTokenGeneratedOnPullRequest() throws Exception {
     MockRM rm1 = new MockRM(conf);
     rm1.start();
     MockNM nm1 = rm1.registerNode("127.0.0.1:1234", 8000);
+    CapacityScheduler cs = getCapacityScheduler(rm1, 8);
```

Comment:
CS object seems to be unused. To avoid further confusion I would separate the getter and the setup method.

Reply:
done